<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_label" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_movement">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>260</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    Mouse Movement</string>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_pressed">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>350</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Pressed</string>
    </property>
   </widget>
   <widget class="Line" name="x_axis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>700</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="y_axis">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>0</y>
      <width>5</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_axes">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>230</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axes</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="circle_radius">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>240</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="set_point1">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>120</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set point 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_point2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>150</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set point 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>200</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_grid">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Grid</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="grid_size">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>50</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="bresenham_line">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>82</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bresenham</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dda">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>42</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DDA</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>1096</x>
      <y>580</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Elapsed</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_show">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>610</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="line_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>13</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Line Algorithms</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="circle_label">
    <property name="geometry">
     <rect>
      <x>714</x>
      <y>128</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle Algorithms</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="polar_circle">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>156</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Polar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bresenham_cicle">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>196</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bresenham</string>
    </property>
   </widget>
   <widget class="QLabel" name="radius">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius</string>
    </property>
   </widget>
   <widget class="QLabel" name="ellipse_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>272</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ellipse Algorithms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polar_ellipse">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>299</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Polar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bresenham_ellipse">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bresenham</string>
    </property>
   </widget>
   <widget class="QLabel" name="radius_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius-X</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="radius_x">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>390</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="radius_y">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>420</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="radius_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius-Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanline">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>460</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ScanLine-Fill</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1219</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header location="global">my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
